app/views/judgements/_form.html.erb

```diff
@@ -405,9 +414,18 @@
     </div>
     <div class="field">
       <% @document_fields.keys.each do |key| %>
-        <div class="field">
-          &nbsp;&nbsp;&nbsp;<strong><%= key %></strong> <%= @document_fields[key] %>
-        </div>
+        <% if key == "title" %>
+          <div class="field">
+            &nbsp;&nbsp;&nbsp;<strong><%= key %></strong> <%= @document_fields[key] %>
+          </div>
+        <% end %>
+      <% end %>
+      <% @document_fields.keys.each do |key| %>
+        <% if key != "title" %>
```

Comment:
change this to a if else inside the loop....

Reply:
do a pluck....    to grab the title out...   select_if or select_first...  look at Enumerable...